<?xml version="1.0"?>
<launch>

    <arg name="follower"                default="pure_pursuit"  doc="pure_pursuit, stanley" />

    <group ns="autoware_mini">

        <!-- Follower -->
        <include file="$(find autoware_mini)/launch/control.launch">
            <arg name="follower" value="$(arg follower)" />
        </include>

        <!-- Bicycle_model simulator -->
        <node pkg="autoware_mini" type="bicycle_simulation.py" name="bicycle_simulation" output="screen" required="true"/>

        <!-- Lanelet2 map loader -->
        <node pkg="autoware_mini" type="lanelet2_map_loader.py" name="lanelet2_map_loader" output="screen" required="true"/>

        <!-- Lanelet2 global planner -->
        <node pkg="autoware_mini" type="lanelet2_global_planner.py" name="lanelet2_global_planner" output="screen" required="true"/>

        <!-- Rviz -->
        <node type="rviz" name="rviz" pkg="rviz" args="-d $(find autoware_mini)/rviz/autoware_mini_sim.rviz" required="true"/>
    </group>

</launch>